### PR TITLE
chore: switch playwright tests to use playwright test runner instead of vitest

### LINKED ISSUE #8630: Switch from vitest test runner to playwright

### PR DESCRIPTION
**Pull request by cbr7**
### What does this PR do?
switches e2e tests to run on playwright test runner instead of vitest

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop/issues/8630

### CODE REVIEW

File: tests/playwright/src/specs/compose-onboarding-smoke.spec.ts [hunk 1]
```diff
@@ -16,11 +16,8 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import { afterEach } from 'node:test';
```
Comment (benoitf) [inline tests/playwright/src/specs/compose-onboarding-smoke.spec.ts:19]: interesting here, it means that the afterEach is probably never called today

Comment (cbr7) [reply tests/playwright/src/specs/compose-onboarding-smoke.spec.ts:19]: why would that be?

Comment (benoitf) [reply tests/playwright/src/specs/compose-onboarding-smoke.spec.ts:19]: vitest analyze only the vitest afterEach

Comment (cbr7) [reply tests/playwright/src/specs/compose-onboarding-smoke.spec.ts:19]: oh I see, that is interesting.

**Comment by cbr7** [comment]
Closing PR for now to not spam the cicd infrastructure.

**Comment by cbr7** [comment]
I also ran the full e2e test set [here](https://github.com/containers/podman-desktop/actions/runs/10630075870/job/29468239099)

**Comment by benoitf** [comment]
@cbr7 could you rebase your branch, there was an issue in the website build that is now fixed

**Review by benoitf — APPROVED** [review]

**Comment by odockal** [comment]
@cbr7 On the first look it looks good. I am interested in the running the new test runner in windows and mac, have you tried?

**Comment by cbr7** [comment]
@odockal I tried but it didn't work because the windows workflows are still using yarn instead of pnpm so it throws an error.

Unless you mean locally, in which case yes, I ran them on my macbook.

**Pull request merged**